src/templates/basicDocPageTemplate.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+
+const basicDocPageTemplate = ({ data }) => {
+  const { mdx } = data;
+  const { frontmatter, body } = mdx;
+
+  return (
+    <>
+      <h1>{frontmatter.title}</h1>
+      <div>{body}</div>
```

Comment:
So it renders the .mdx file body content
```suggestion
        <MDXProvider>
          <MDXRenderer>{body}</MDXRenderer>
        </MDXProvider>
```